### ISSUE TITLE
Sometimes the crate cannot be built

### ISSUE THREAD

**Issue by Luni-4** [evenorog/license#5]
Because of this error, that only sometimes occurs, it is difficult to build the crate

```sh
error: failed to run custom build command for `license v3.2.0`

Caused by:
  process didn't exit successfully: `/home/target/debug/build/license-5e8ba4d961a88bb4/build-script-build` (exit status: 101)
  --- stderr
  Cloning into 'license-list-data'...
  fatal: unable to access 'https://github.com/spdx/license-list-data.git/': Failed to connect to github.com port 443 after 138630 ms: Couldn't connect to server
  thread 'main' panicked at /home/.cargo/registry/src/index.crates.io-6f17d22bba15001f/license-3.2.0/build.rs:103:35:
  `git clone --depth 1` failed with exit code `128`
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by evenorog** [comment]
Hi, looks like it can't connect to github.
You can use the `offline` feature to fix this for now, and I will try to fix this in #3 later.

**Comment by Luni-4** [comment]
Up till now the `offline` feature seems to work really well, thanks!
